packages/cloudflare/src/api/d1-tag-cache.ts

```diff
@@ -0,0 +1,151 @@
+import { debug, error } from "@opennextjs/aws/adapters/logger.js";
+import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
+import type { TagCache } from "@opennextjs/aws/types/overrides.js";
+
+import { getCloudflareContext } from "./cloudflare-context.js";
+
+// inlined during build
+const manifest = process.env.__OPENNEXT_CACHE_TAGS_MANIFEST;
+
+class D1TagCache implements TagCache {
+  public name = "d1-tag-cache";
+
+  public async getByPath(rawPath: string): Promise<string[]> {
+    const { isDisabled, db, table } = this.getConfig();
+    if (isDisabled) return [];
+
+    const path = this.getCacheKey(rawPath);
+
+    try {
+      const { success, results } = await db
+        .prepare(`SELECT tag FROM ${table} WHERE path = ?`)
```

Comment:
can't we use bind for the table name? (SQL injection)

Reply:
Unfortunately not - it gives a syntax error. The table is driven by an env var, so i wouldnt think sql injection will be a problem

Reply:
It should not... but maybe we can hardcode `tags` here anyway.

Reply:
I did this with the idea that people might want to use a different table name (multiple projects using same db), so it was using the env var and defaulting to `tags` if not present. I can change to always hardcode to tags if you want, but I feel like it's a useful bit of a configurability to have?